desktop/assets/Assets.xcassets/RuffleMacIcon.iconset/Contents.json

```diff
@@ -0,0 +1,68 @@
+{
```

Comment:
As I'm unfamiliar with macOS, I may be wrong, but Apple's documentation points out that `Contents.json` is not needed for `.iconset`: https://developer.apple.com/library/archive/documentation/Xcode/Reference/xcode_ref-Asset_Catalog_Format/AssetTypes.html#//apple_ref/doc/uid/TP40015170-CH30-SW5

Reply:
I actually switched between `.iconset` and `.appiconset` multiple times when fighting with the asset compiler. AFAIK `.appiconset` is actually for iOS only, though.

The real reason why that file got in there, though, is because Affinity Designer handed me it when I selected the app icon export option in it's slicer. Said exporter also caused the problems I mentioned above with the asset compiler (since Apple mixes 1x and 2x assets in the PSD template they provide for macOS app icons).